src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewaySettings.cs

```diff
@@ -20,6 +20,8 @@ public sealed class FederationGatewaySettings : IFederationGatewaySettings
 
         private const string FederationIpsParam = "federationips";
 
+        private const string MinimumDepositConfirmationsParam = "minimumdepositconfirmations";

```

Comment:
Does this get used anywhere?

Reply:
oh yes correct, I removed the rest of the logic as this shouldn't be changeable at the federation member level, that would be chaotic.